crypto/hashes/src/lib.rs

```diff
@@ -3,14 +3,15 @@ mod pow_hashers;
 
 use serde::{Deserialize, Serialize};
 use std::fmt::{Debug, Display, Formatter};
+use std::hash::{Hash as StdHash, Hasher as StdHasher};
 use std::str::{self, FromStr};
 
 pub const HASH_SIZE: usize = 32;
 
 pub use hashers::*;
 
 // TODO: Check if we use hash more as an array of u64 or of bytes and change the default accordingly
```

Comment:
Is this still a TODO?

Reply:
Yes, when the code is "complete" we should check which methods are used the most